mantidimaging/gui/windows/spectrum_viewer/model.py

```diff
@@ -208,6 +208,17 @@ def normalise_issue(self) -> str:
             return "Stack shapes must match"
         return ""
 
+    def shuttercount_issue(self) -> str:
+        """
+        Return an error message if there is an issue with the shutter count data.
+        """
+        assert self._stack and self._normalise_stack is not None
```

Comment:
Should that be `assert self._stack is not None and self._normalise_stack is not None`